js/config.js

```diff
@@ -13,6 +13,8 @@ const DEFAULT_CONFIG = {
   playing: true,
   // Save model state every N steps. It can be restored later.
   snapshotInterval: 100,
+  // How many steps between plate centers are recalculated.
+  centerUpdateInterval: 15,
```

Comment:
It might be a matter of taste, but I'd say that values around 60-120 look better actually. Labels move less and seem more stable. Especially in these final moments before plate disappears or in the `circles` test.
Also, not important here, but I'd use `config` for values that Sarah or other authors might be interested in setting (I guess this one is not likely), otherwise just local constants in given file.